wired_module/wired_module_arduino/src/main.cpp

```diff
@@ -0,0 +1,95 @@
+#include <WiFi.h>
+#include <PubSubClient.h>
+
+#include <BarometerSensor.h>
+#include <I2cMaster.h>
+#include <MpuSensor.h>
+#include <Adc1.h>
+# include <GpsSensor.h>
+# include <ICM42670.h>
+
+# include <vector>
+# include "driver/i2c.h"
+# include "DPS368.cpp"
+
+// Wi-Fi and MQTT Settings
+const char* ssid = "A 2 jang pura extn_5G";
+const char* password = "act12345";
+const char* mqttServer = "192.168.0.117";  // will replace it later with raspberry's IP
```

Comment:
These shouldn't be written and committed directly into the main file. Instead we should use a `.env` or some similar type of configuration file. The contents of this file in this case will contain something like the below:
``` 
SSID="some_SSID"
PASSWORD="some_password"
MQTT_SERVER="ip_for_server"
```

The `main.cpp` will read from this file, the file should also be `gitignored` so we don't put these secrets onto Git. This way for anyone's test environment they can play around with the values to make it work and it keeps our data confidential. Let me know if you have any questions about this.